app/Views/errors/html/debug.css

```diff
@@ -1,197 +1,197 @@
 :root {
-	--main-bg-color: #fff;
-	--main-text-color: #555;
-	--dark-text-color: #222;
-	--light-text-color: #c7c7c7;
-	--brand-primary-color: #E06E3F;
-	--light-bg-color: #ededee;
-	--dark-bg-color: #404040;
+    --main-bg-color: #fff;
+    --main-text-color: #555;
```

Comment:
Are these diffs generated by Dart SASS?

Reply:
Oh, it seems my mistake.

Reply:
No, it is not generated by SASS.
I replaced tabs with spaces.

Reply:
Ok. It seems kinda weird that they look the same from my PC.

![image](https://user-images.githubusercontent.com/51850998/170666738-7a397aaa-330c-4b03-bb8d-5d7de1f1f2b1.png)